scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -117,7 +117,8 @@ task CoreStorageModelSizes {
             local table_pattern="$1"
             local output_file_name="$2"
 
-            bq --apilog=false query --project_id='~{project_id}' --format=csv --use_legacy_sql=false \
+            # bq query --max_rows check: explicitly set massive max rows as we expect there to be as many rows as there are superpartitions
```

Comment:
isn't this going to always return one line because it's a `sum()`?

Reply:
oops yes